lib/components/Heading/Heading.migration.md

```diff
@@ -0,0 +1,17 @@
+# Heading Migration Guide
+
+## API Changes
+
+- Extracted from `Text` into a standalone component.
+- Deprecated all existing size and weight variants in favour of `level={'1' | '2' | '3'}` and `weight={'regular' | 'weak'}`. Please note that this is a breaking change to our design language. When migrating, try to match existing styling as much as possible in consultation with your local designer.
+- Removed `baseline={boolean}`.
```

Comment:
Also adds whitespace below, so consumers will need to validate their usage, may require deleting styles from their app (from what i have observed)